### ISSUE TITLE
Can not connect to ws://a0d7b954-zwavejs2mqtt:3000

### ISSUE THREAD

**Issue by koburg** [hassio-addons/addon-zwavejs2mqtt#278]
# Problem/Motivation
installing new vm from HASS (google backup) backup

## Expected behavior

connect to ws://a0d7b954-zwavejs2mqtt:3000

## Actual behavior

getting error message never experienced before normally just work
<img width="396" alt="0932350A-F77E-424B-B24B-5BBEABDAA921" src="https://user-images.githubusercontent.com/3473374/139587307-235b4d53-0802-4bab-a786-c86fe921149c.png">

## Steps to reproduce

after restore to new vm from backup  homeassistant z-wavejs says not able to connect (picture)
when typing the ws://a0d7b954-zwavejs2mqtt:3000

so i deleted the integration and tried again and again....
no luck

> (How can someone else make/see it happen)
don't know because never been this issue before

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

running vm virtual box never exprienced this before.

**Comment by frenck** [comment]
Anything in the add-on logs?

**Comment by koburg** [comment]
not really only thing is this don't know if it is to any use. 

`2021-11-02 16:56:49.189 INFO APP: GET / 304 1.207 ms - -
2021-11-02 16:56:49.210 INFO APP: GET /static/js/app.433ebbae6250eed321a0.js 304 1.612 ms - -
2021-11-02 16:56:49.222 INFO APP: GET /static/css/app.c3c029fa61114f4f5b2f.css 304 1.810 ms - -
2021-11-02 16:56:49.454 INFO APP: GET /api/auth-enabled 304 0.987 ms - -
2021-11-02 16:56:49.489 INFO APP: GET /static/logo.png 304 0.474 ms - -
2021-11-02 16:56:49.673 INFO APP: GET /static/fonts/MaterialIcons-Regular.fa3334f.woff2 304 1.033 ms - -
2021-11-02 16:56:49.761 INFO APP: GET /api/settings 304 79.894 ms - -
2021-11-02 16:56:49.791 INFO APP: GET /api/auth-enabled 304 1.034 ms - -`

**Comment by frenck** [comment]
I cannot do anything with that.

**Comment by koburg** [comment]
It is working now BUT!!

I needed to uninstall zwave2mqtt shutdown completely HASS (first tried to uninstall and reinstall with restart from ui not working) restart Hass reinstall z2m and now I need to start over place all my devices in rooms etc. 

I also did upgrade to just released 2021.11 from 2021.10.xx

**Comment by circa1741** [comment]
Just sharing my experience as it might provide some additional information.

What I have tried:

1. zwave2mqtt configured with Start on boot = true and Watchdog = true. I updated zwave2mqtt from 0.27.0 to 0.28.0. HA would not reconnect.

2. I reconfigured Start on boot = false and Watchdog = false. I updated zwave2mqtt from 0.27.0 to 0.28.0. HA reconnects and zwave2mqtt was off because Start on boot = false. However, when I started zwave2mqtt, it would eventually disconnect and would not reconnect.

I am hoping I don't have to do what koburg did. Uninstall zwave2mqtt, reboot HA, reinstall zwave2mqtt then reconfigure everything.
